### ISSUE TITLE
[Testing] Computing the Accuracy Performance with Pre-Trained COCO128 Dataset and YOLO on Server, Raspberry Pi 4, and Jetsan Nano

### ISSUE THREAD

**Issue by jaskiratsingh2000** [jaskiratsingh2000/Research-Scalable-Vehicle-Detection-on-Edge-Devices#19]
**Accuracy Computed on Server**

```
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|████████████████████| 4/4 [00:03<00:00,  1.05it/s]
                 all        128        929      0.721      0.692       0.76      0.484
Speed: 0.3ms pre-process, 7.1ms inference, 2.8ms NMS per image at shape (32, 3, 640, 640)
Results saved to runs/test/exp
```

**Comment by jaskiratsingh2000** [comment]
**Accuracy Computed on Raspberry Pi 4**

```
               Class     Images     Labels          P          R     mAP@.5
                 all        128        929   3.53e-05    0.00201   6.39e-05   6.39e-06
Speed: 12.4ms pre-process, 3827.1ms inference, 3.3ms NMS per image at shape (32, 3, 640, 640)
Results saved to runs/test/exp4
```

**Comment by jaskiratsingh2000** [comment]
Accuracy Computed on Raspberry Pi 4 with batch 16 epochs 5
```
               Class     Images     Labels          P          R     mAP@.5
                 all        128        929    0.00251    0.00348   0.000251   2.51e-05
Speed: 12.3ms pre-process, 1812.5ms inference, 5.5ms NMS per image at shape (32, 3, 640, 640)
```

**Comment by jaskiratsingh2000** [comment]
Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|████████████████████| 4/4 [00:03<00:00,  1.08it/s]
                 all        128        929      0.675      0.627      0.672      0.447
Speed: 0.2ms pre-process, 3.3ms inference, 3.0ms NMS per image at shape (32, 3, 640, 640)
Results saved to runs/test/exp
